Tests/EosioSwiftTests/EosioTransactionTests.swift

```diff
@@ -82,7 +82,7 @@ class EosioTransactionTests: XCTestCase {
 
     func test_prepare_shouldCallGetBlockFunctionOfRPCProvider() {
         transaction.prepare { (_) in
-            XCTAssertTrue(self.rpcProvider.getBlockInfoCalled)
+            XCTAssertTrue(self.rpcProvider.getBlockInfoCalled || self.rpcProvider.getBlockCalled)
```

Comment:
I'd like it if we could have a test run where the mocked `getInfo` returns the current v1.3.0 value and then v2.0.0, 2.1.0-rc1, and 2.1.0 just to show that we've covered all those bases.  I changed it manually myself and all worked, so I know its good but I'd like the test to reflect that as well.  Thanks!